docs/source/tracking/artifacts-stores.rst

```diff
@@ -0,0 +1,200 @@
+===============
+Artifact Stores
+===============
+
+The artifact store is a core component in `MLflow Tracking <../index.html>`_ where MLflow stores (typicaly large) arifacts
+for each run such as model weights (e.g. a pickled scikit-learn model), images (e.g. PNGs), model and data files (e.g. `Parquet <https://parquet.apache.org/>`_ file). 
+Note that metadata like parameters, metrics, and tags are stored in `backend store <backend-stores.html>`_, the other component of the MLflow Tracking.
+
+Configure Artifact Store
+========================
+MLflow by default stores artifacts in local ``./mlruns`` directory, but also supports various locations suitable for large data:
+AAmazon S3, Azure Blob Storage, Google Cloud Storage, SFTP server, and NFS. You can connect those remote storages via MLflow Tracking server.
+See :ref:`tracking server setup <tracking-server-artifact-store>` and the specific section for your storage in :ref:`supported storages <artifacts-store-supported-storages>` for 
+how to connect to the remote storage.
+
+.. _artifacts-stores-manage-access:
+
+Manage Access
+-------------
+To allow the server and clients to access the artifact location, you should configure your cloud
+provider credentials as normal. For example, for S3, you can set the ``AWS_ACCESS_KEY_ID``
+and ``AWS_SECRET_ACCESS_KEY`` environment variables, use an IAM role, or configure a default
+profile in ``~/.aws/credentials``.
+
+.. important::
+    Access credentials and configuration for the artifact storage location are configured *once during server initialization* in the place
+    of having users handle access credentials for artifact-based operations. Note that *all users who have access to the
+    Tracking Server in this mode will have access to artifacts served through this assumed role*.
+
+Timeout
+-------
+You can set an environment variable ``MLFLOW_ARTIFACT_UPLOAD_DOWNLOAD_TIMEOUT`` (in seconds) to configure the timeout for artifact uploads and downloads.
+Note that this is experimental feature, may be changed or removed.
```

Comment:
```suggestion
Note that this is an experimental feature which may be changed or removed.
```
Can we indicate what the default value is for this environment variable so that users don't have to look it up in source code?

Reply:
Makes sense! fyi apparently this one follows default timeout for each storage provider.